MCInst.h

```diff
@@ -131,7 +131,7 @@ struct MCInst {
 	uint8_t xAcquireRelease;   // X86 xacquire/xrelease
 	bool isAliasInstr; // Flag if this MCInst is an alias.
 	bool fillDetailOps; // If set, detail->operands gets filled.
-	hppa_ext hppa_ext;
+	hppa_ext hppa_ext;	// for HPPA operand
```

Comment:
```suggestion
	hppa_ext hppa_ext;	///< for HPPA operand
```
This is some kind of extensions of immediate values? Please clarify in the doc string.